pdf/rotate.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package pdf
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	unicreator "github.com/unidoc/unidoc/pdf/creator"
```

Comment:
Not sure if the "uni" is needed, "creator" should be pretty clear in the context within the pdf package.

Reply:
Yes, the prefix is not needed in this file. However, every import from unidoc throughout the application has this prefix. In my opinion, we should continue to use it like this for consistency.